client/src/app.jsx

```diff
@@ -1,8 +1,8 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-// import ProductOverview from './components/ProductOverview.jsx'
-// import RelProducts from './components/RelProducts.jsx'
-// import QnA from './components/QnAcomponents/mainQnA.jsx';
+import ProductOverview from './components/ProdOverview/OverView.jsx'
+import RelProducts from './components/RelProducts.jsx'
```

Comment:
./components/RelProductsComponents/RelProducts.jsx

Need the folder name in between